### PR TITLE
Remove actor

### PR DESCRIPTION
**Pull request by DmytroOrlov**

### CODE REVIEW

File: server/app/controllers/TaskSolver.scala [hunk 1]
```diff
@@ -62,7 +61,7 @@ class TaskSolver @Inject()(dynamicExecutor: DynamicSuiteExecutor, runtimeExecuto
     val channel: Observable[Event] =
       Observable.create[Event](OverflowStrategy.DropOld(20)) { downstream =>
         val cancelable = AssignableCancelable.single()
-        clientInputPromise.future.onComplete {
+        clientInputPromise.future.timeout(1.second).onComplete {
```
Comment (novakov-alexey) [inline server/app/controllers/TaskSolver.scala]: what does it mean 1 second timeout here?

Comment (DmytroOrlov) [inline server/app/controllers/TaskSolver.scala]: Server will wait for client input no longer than 1 second

File: server/app/Module.scala [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import com.google.inject.AbstractModule
+import com.google.inject.name.Names.named
+
+import scala.util.Random.nextInt
+
+class Module extends AbstractModule {
+  override def configure() =
```
Comment (novakov-alexey) [inline server/app/Module.scala]: I do not like the idea with ImplemenetedBy annotation. Now, in order to see which components I create with dependency injection framework, I need to run over all classes in project and see what it is available. Having this binding inside the Module class is much better. There is no significant win to use annotations. Why did you move this to annotations?

Comment (DmytroOrlov) [inline server/app/Module.scala]: I agree, but in case if `ImplemenetedBy` is in the same file as implemetation it is ok for me. You don't need to
>run over all classes in project

Don't you?

Comment (novakov-alexey) [inline server/app/Module.scala]: Right, but if I want to see all components then I would need to do that

Comment (DmytroOrlov) [inline server/app/Module.scala]: Agree with you, but in such micro applications it's not a problem, can revert `AbstractModule.configure()` easily

Comment (novakov-alexey) [inline server/app/Module.scala]: yes, as long as it is small application
